tests/regression/76-memleak/09-unreachable-with-local-var.c

```diff
@@ -0,0 +1,17 @@
+//PARAM: --set ana.malloc.unique_address_count 1 --set ana.activated[+] memLeak
+#include <stdlib.h>
+
+int *g;
+
+int main(int argc, char const *argv[]) {
+    g = malloc(sizeof(int));
+    // Reference to g's heap contents is lost here
+    g = NULL;
+    // We get a false positive for p's memory being unreachable
+    // It's still leaked, but due to free() being commented out
+    // TODO: Should we only improve the error reporting for unreachable memory in this case?
```

Comment:
At least for `valid-memtrack` this is not actually a false-positive.

Reply:
Ah, yes. I saw from the discussion in https://gitlab.com/sosy-lab/benchmarking/sv-benchmarks/-/merge_requests/1464 that if the memory is reachable from locals then it's a correct reporting of a `valid-memtrack` violation.

I'll remove the `// TODO` in this case.